go.mod

```diff
@@ -18,11 +18,11 @@ require (
 	github.com/siderolabs/go-blockdevice v0.4.8
 	github.com/siderolabs/image-factory v0.5.0
 	github.com/siderolabs/net v0.4.0
-	github.com/siderolabs/talos v1.8.1
-	github.com/siderolabs/talos/pkg/machinery v1.8.1
+	github.com/siderolabs/talos v1.9.0-alpha.0
+	github.com/siderolabs/talos/pkg/machinery v1.9.0-alpha.0
 	github.com/stretchr/testify v1.9.0
 	golang.org/x/mod v0.21.0
-	google.golang.org/grpc v1.66.3
+	google.golang.org/grpc v1.67.1
```

Comment:
this might be tricky... we'll hit the ALPN issue https://github.com/siderolabs/talos/issues/9463

We need to deal with that anyways in the provider, otherwise it won't work with Talos < 1.9

I'm going to backport compatibility for 1.9 into 1.8.2, probably we should wait for it?

Reply:
I didn't intent to bump since last time I broke it, reverting back

Reply:
ahh it's from talos machinery the bump happended